src/shared/components/EditTableForm.tsx

```diff
@@ -94,6 +95,8 @@ const EditTableForm: React.FC<{
   busy,
   formName = 'Edit',
 }) => {
+  const editorRef = React.useRef<codemirror.Editor>();
+  const wrapperRef = React.useRef(null);
   // state
```

Comment:
Can this be removed?

Reply:
can not be removed, they are the important thing here 😉 
look at `editorDidMount` and `editorWillUnmount` in the `<CodeMirror/>` component